### DISCUSSION TITLE
CART3 Turbine Characteristics

### DISCUSSION THREAD

**Discussion by KarimMohammadd** [OpenFAST/openfast#1131] [Q&A]
Dear all,
Does anyone know where can I find complete characteristics of the CART3 turbine (such as blade inertia about shaft, blade stiffness, generator inertia, and …)? I need these data for FAST simulation of CART3.
Thank you.

**Comment by jjonkman** [comment]
Dear @KarimMohammadd,

Sorry, but the CART2 and CART3 FAST / OpenFAST models are not available in the public domain, as discussed a bit in the following topic on our forum: http://forums.nrel.gov/t/help-running-fast-v8-for-the-first-time/1205/6.

Best regards,
